pkg/pillar/types/zedroutertypes.go

```diff
@@ -1594,6 +1695,17 @@ func IsPort(globalStatus DeviceNetworkStatus, ifname string) bool {
 	return false
 }
 
+// IsEndpoint checks if an interface name belongs to a port with SystemAdapter attached.
+func IsEndpoint(globalStatus DeviceNetworkStatus, ifname string) bool {
```

Comment:
The name IsEndpoint could be more clearly indicate it is a subset of the IsPorts.
Can we name it IsEndpointPort, or IsL3Port, or something else with "Port" in the name.`

Reply:
Renamed to `IsL3Port` which sounds better.